### PR TITLE
Scheduler Custom Templates: rewrite

### PR DESCRIPTION
**Pull request by pavel-2020**
[Trello](https://trello.com/c/ggGmCBjS/1669-scheduler-demos-customization-3-demos)
[Demo](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Scheduler/CustomTemplates/jQuery/Light/)
CPs: https://github.com/DevExpress/devextreme-demos/pull/960, https://github.com/DevExpress/devextreme-demos/pull/961

### CODE REVIEW

File: JSDemos/Demos/Scheduler/CustomTemplates/description.md [hunk 1]
```diff
@@ -1 +1,7 @@
-This demo illustrates the rich collection of&nbsp;the appearance customization properties available as&nbsp;a&nbsp;part of&nbsp;the Scheduler component. The appearance of&nbsp;appointments and appointment tooltips are defined using custom templates. Properties for the appointment details form are set within the **appointmentFormOpening** event handler. Properties for the Scheduler are also modified: the **timelineDay** view is&nbsp;added to&nbsp;the _View Selector_ and the _All-day panel_ is&nbsp;hidden.
\ No newline at end of file
+This demo shows the following Scheduler properties that specify custom templates:
+
+* [appointmentTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#appointmentTemplate) / **views[]**.[appointmentTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#appointmentTemplate)
```
Comment (AlexJameson) [inline JSDemos/Demos/Scheduler/CustomTemplates/description.md]: Why did you decide to omit the descriptions? How can users understand the difference between these properties? 

Also, could you please show in which line we demonstrate the **views[]**.appointementTemplate property? I guess that it should be used to customize an individual view but I can't see it anywhere. 

The same notes apply to the next list item.

File: JSDemos/Demos/Scheduler/CustomTemplates/description.md [hunk 2]
```diff
@@ -1 +1,7 @@
...
+
+* [appointmentTooltipTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#appointmentTooltipTemplate) / **views[]**.[appointmentTooltipTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#appointmentTooltipTemplate).
+
+To customize the appointment details form, implement the [onAppointmentFormOpening](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#onAppointmentFormOpening) handler. In this demo, we use it to add custom fields.
```
Comment (AlexJameson) [inline JSDemos/Demos/Scheduler/CustomTemplates/description.md]: Let;s get rid of the first person plural here. Maybe something like that? 

In this demo, this handler adds custom fields to the appointment details form.

**Review by AlexJameson — DISMISSED** [review]

**Review by RomanTsukanov — DISMISSED** [review]

File: JSDemos/Demos/Scheduler/CustomTemplates/description.md [hunk 3]
```diff
@@ -1 +1,7 @@
-This demo illustrates the rich collection of&nbsp;the appearance customization properties available as&nbsp;a&nbsp;part of&nbsp;the Scheduler component. The appearance of&nbsp;appointments and appointment tooltips are defined using custom templates. Properties for the appointment details form are set within the **appointmentFormOpening** event handler. Properties for the Scheduler are also modified: the **timelineDay** view is&nbsp;added to&nbsp;the _View Selector_ and the _All-day panel_ is&nbsp;hidden.
\ No newline at end of file
+Scheduler offers the following properties to specify custom templates globally and for individual views:
```
Comment (NikolaevaIrina) [inline JSDemos/Demos/Scheduler/CustomTemplates/description.md]:
```suggestion
Scheduler contains the following properties to specify custom templates globally and for individual views:
```

File: JSDemos/Demos/Scheduler/CustomTemplates/description.md [hunk 4]
```diff
@@ -1 +1,7 @@
...
+
+* Appointment rectangle: [appointmentTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#appointmentTemplate) / **views[]**.[appointmentTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#appointmentTemplate)
+
+* Tooltip: [appointmentTooltipTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#appointmentTooltipTemplate) / **views[]**.[appointmentTooltipTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#appointmentTooltipTemplate).
+
+To customize the appointment details form, implement the [onAppointmentFormOpening](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#onAppointmentFormOpening) handler. In this demo, this handler adds custom fields to the appointment details form.
```
Comment (NikolaevaIrina) [inline JSDemos/Demos/Scheduler/CustomTemplates/description.md:7]: В Blazor документации мы чаще всего используем implement с interface. Поэтому, я бы заменила implement (в этом контексте) на use/create/add. Если же у вас в доке так пишется, то я не имею ничего против :)

File: JSDemos/Demos/Scheduler/CustomTemplates/description.md [hunk 5]
```diff
@@ -1 +1,7 @@
-This demo illustrates the rich collection of&nbsp;the appearance customization properties available as&nbsp;a&nbsp;part of&nbsp;the Scheduler component. The appearance of&nbsp;appointments and appointment tooltips are defined using custom templates. Properties for the appointment details form are set within the **appointmentFormOpening** event handler. Properties for the Scheduler are also modified: the **timelineDay** view is&nbsp;added to&nbsp;the _View Selector_ and the _All-day panel_ is&nbsp;hidden.
\ No newline at end of file
+Scheduler contains the following properties to specify custom templates globally and for individual views:
```
Comment (arminal) [inline JSDemos/Demos/Scheduler/CustomTemplates/description.md]:
```suggestion
The Scheduler contains the following properties used to specify custom templates globally and for individual views:
```

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**